test/controllers/home_controller_test.rb

```diff
@@ -1,7 +1,20 @@
 require 'test_helper'
 
 class HomeControllerTest < ActionController::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @user = User.create!(email: "test@test.com", password: "testpassword",
+                         first_name: "Test", last_name: "Test")
+  end
+
+  test 'should redirected to items' do
+    login(@user)
+
+    get :index
+    assert_redirected_to items_path
+  end
+
+  test 'when not login, should redirected to login' do
+    get :index
+    assert_redirected_to login_path
+  end
```

Comment:
ah looks like it's here - https://github.com/glaksmono/moka-rails-interview/pull/4/files#diff-6d313d1fc37182e0aa18b6b426cf2619R21